server/views/pages/reports.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "moj/components/search/macro.njk" import mojSearch -%}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = makePageTitle("Reports", errors) %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {% if errors %}
+                {{ govukErrorSummary({
+                    titleText: "There is a problem",
+                    errorList: errors.list,
+                    attributes: {
+                        'data-qa': 'error-list'
+                    }
+                }) }}
+            {% endif %}
+            <h1 class='govuk-heading-l'>Reports</h1>
+
+            <p class='govuk-body-l'>Click on a report to download it.</p>
+            <div{% if errors.error_WEEKLY_RECALLS_NEW %} id='downloadLinkError'{% endif %}>
+            {{ documentDownloadLink({
+                linkText: "Weekly Recalls New",
```

Comment:
"Weekly New Recalls" reads a bit better?

Reply:
Yes; we can change of course; but this is the report name as per the story.  In time we'll get feedback from Product team of course.